aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -61,10 +61,30 @@ namespace native {
 template<typename scalar_t>
 scalar_t dot_impl(int64_t n, scalar_t *x, int64_t incx, scalar_t *y, int64_t incy);
 
-static void make_offset2bag(const Tensor &offsets, Tensor& offset2bag) {
-  offset2bag.index_add_(
```

Comment:
Changed to use `narrow` instead of `clone`.

Anyway, I am pretty sure that `index_add` would go sequential at this situation (dimension of the input tensor is 1), so the above code should not be slower. But I also want to make it simpler otherwise it would be difficult for debug. This file is already very complexed ...